Doc/library/constants.rst

```diff
@@ -94,6 +96,13 @@ should not be used in programs.
    (i.e. EOF) to exit", and when called, raise :exc:`SystemExit` with the
    specified exit code.
 
+.. data:: help
+   :noindex:
+
+   Object that when printed, prints the message "Type help() for interactive
+   help, or help(object) for help about object.", and when called, invokes the
```

Comment:
The first clause, giving the special effect of `help`, is not part of the 'help' entry in Built-in Functions. For this reason, and completion otherwise, I think this entry should be added.